### PR TITLE
corrections and new categories related to yogurts

### PR DESCRIPTION
**Pull request by moon-rabbitOFF**

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -38219,7 +38240,7 @@ fr:Spaghetti, spaghettis
 hu:Spagetti
 it:Spaghetti
 nl:Spaghettis
```
Comment (aleene) [inline taxonomies/categories.txt]: nl:Spaghetti, Spaghettis

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -38219,7 +38240,7 @@ fr:Spaghetti, spaghettis
 hu:Spagetti
```
Comment (aleene) [inline taxonomies/categories.txt]: Issue here with lemon soy yoghurts and spaghetti.

nl:Citroensojayoghurt, Citroensojayoghurts, Sojayoghurt met citroen

File: taxonomies/categories.txt [hunk 3]
```diff
@@ -24841,6 +24863,7 @@ it:Yogurt al cocco
 nl:Yoghurt met kokos
 
 <en:Fruit yogurts
+en:Hazelnut yogurts
 fr:Yaourts aux noisettes, Yaourt aux noisettes
 it:Yogurt alla nocciola
```
Comment (aleene) [inline taxonomies/categories.txt]: nl:Hazelnootyoghurt, Hazelnootyoghurts, Yoghurt met hazelnoot

File: taxonomies/categories.txt [hunk 4]
```diff
@@ -24827,7 +24849,7 @@ fr:Yaourts au pamplemousse
 en:Kiwi yogurts
 de:Kiwi-Joghurt
 fr:Yaourts au kiwi
-hu:Kiwis  joghurtok
+hu:Kiwis joghurtok
 it:Yogurt al kiwi
 nl:Yoghurt met kiwi
```
Comment (aleene) [inline taxonomies/categories.txt]: nl:Kiwiyoghurt, Kiwiyoghurts, Yoghurt met kiwi, Yoghurts met kiwi

File: taxonomies/categories.txt [hunk 5]
```diff
@@ -24805,6 +24808,25 @@ fr:Yaourts à la poire
...
+en:Plum yogurts
+es:Yogures con ciruela
+fr:Yaourts à la prune
+it:Yogurt alla prugna, yogurt alla susina
+
+<en:Fruit yogurts
+en:Prune yogurts
+es:Yogures con ciruela pasa
+fr:Yaourts au pruneau
+it:Yogurt alla prugna secca
```
Comment (aleene) [inline taxonomies/categories.txt]: nl:Gedroogde pruimenyoghurt, Gedroogde pruimenyoghurts, Yoghurt met gedroogde pruimen

File: taxonomies/categories.txt [hunk 6]
```diff
@@ -24805,6 +24808,25 @@ fr:Yaourts à la poire
...
+es:Yogures con manzana
+fr:Yaourts à la pomme
+it:Yogurt alla mela
+nl:Yoghurt met appel
+
+<en:Fruit yogurts
+en:Plum yogurts
+es:Yogures con ciruela
+fr:Yaourts à la prune
+it:Yogurt alla prugna, yogurt alla susina
```
Comment (aleene) [inline taxonomies/categories.txt]: nl:Pruimenyoghurt, Pruimenyoghurts, Yoghurt met pruimen

File: taxonomies/categories.txt [hunk 7]
```diff
@@ -24805,6 +24808,25 @@ fr:Yaourts à la poire
 it:Yogurt alla pera
 nl:Yoghurt met peer
 
+<en:Fruit yogurts
+en:Apple yogurts
+es:Yogures con manzana
+fr:Yaourts à la pomme
+it:Yogurt alla mela
+nl:Yoghurt met appel
```
Comment (aleene) [inline taxonomies/categories.txt]: nl:Appelyoghurt, Appelyoghurts, Yoghurt met appel

**Comment by moon-rabbitOFF** [comment]
There is a bug, I can't edit my pull request to add your changes. When I click on "edit file", I get a blank sheet

**Comment by stephanegigandet** [comment]
@aleene: apparently @moon-rabbitOFF is running into issues to add your changes, can you approve the changes so that we can merge, and then we can add the Dutch translations

**Comment by stephanegigandet** [comment]
Thank you!

**Pull request merged**